kumascript/macros/accessibilitySidebar.ejs

```diff
@@ -0,0 +1,168 @@
+        "Accessibility_and_Spacial_Patterns" : "Spacial Patterns",
+        "Understanding_Colors_and_Luminance" : "Colors and Luminance",
+        "Seizure_disorders" : "Seizures and physical reactions",
+    "Resources" : "Resources",
+        "What_is_accessibility" : "What is accessibility?",
+        "HTML" : "HTML: A good basis for accessibility",
+        "CSS_and_JavaScript" : "CSS and JavaScript accessibility best practices",
+        "WAI-ARIA_basics" : "WAI-ARIA basics",
+        "Multimedia" : "Accessible multimedia",
+        "Mobile" : "Mobile accessibility",
+    "ARIA" : "ARIA",
+    "ARIA_guides" : "ARIA guides",
+        "Annotations" : "Annotations",
+        "ARIA_Guides" : "Guides",
+        "ARIA_Live_Regions" : "Live regions",
+        "ARIA_Screen_Reader_Implementors_Guide" : "Screen reader implementors guide",
+        "ARIA_Techniques" : "Using ARIA",
+        "Multipart_labels" : "Multipart labels",
+        "How_to_file_ARIA-related_bugs" : "How to file bugs",
+    "ARIA states and properties": "ARIA states and properties",
+    "ARIA roles": "ARIA roles",
+  }
+});
+
+const sections = [
+  {
+    name: "Accessibility",
+    link: "Accessibility",
+    subsections: [
+      {
+        name: "Guides",
+        url: baseURL,
+        pages: [
+          "Information_for_Web_authors",
+          "Accessibility:_What_users_can_to_to_browse_safely",
+          "An_overview_of_accessible_web_applications_and_widgets",
+          "Keyboard-navigable_JavaScript_widgets",
+          "Mobile_accessibility_checklist",
+          "Understanding_WCAG",
+          "Cognitive_accessibility",
+          "Accessibility_and_Spacial_Patterns",
+          "Understanding_Colors_and_Luminance",
+          "Seizure_disorders",
+        ]
+      },
+      {
+        name: "Resources",
+        url: LearnURL,
+        pages: [
+          "What_is_accessibility",
+          "HTML",
+          "CSS_and_JavaScript",
+          "WAI-ARIA_basics",
+          "Multimedia",
+          "Mobile",
+        ]
+      }
+    ]
+  },
+  {
+    name: "ARIA",
+    link: "ARIA",
+    subsections: [
+      {
+        name: "ARIA_guides",
+        url: AriaURL,
+        pages: [
+          "Annotations",
+          "ARIA_Guides",
+          "ARIA_Live_Regions",
+          "ARIA_Screen_Reader_Implementors_Guide",
+          "ARIA_Techniques",
+          "Multipart_labels",
+          "How_to_file_ARIA-related_bugs",
+        ]
+      },
+      {
+        name: "ARIA states and properties",
+        url: AriaURL,
+        subdirectory: "attributes",
+      },
+      {
+        name: "ARIA roles",
+        url: AriaURL,
+        subdirectory: "roles",
+      }
+    ]
+  }
+];
+
+async function getTitle(pageSlug, URL) {
+  let baseURL = URL? URL : baseURL;
+  let page = await wiki.getPage(`${baseURL}${pageSlug}`);
+  if (!page.title) {
+    page = await wiki.getPage(`/en-US${sidebarURL}${pageSlug}`);
+  }
+  return mdn.htmlEscape(page.title);
+}
+
```

Comment:
I _think_ we don't need this:

```suggestion
```

Reply:
was needed for the previous `renderSubsection` function. That was updated so this is indeed no longer needed. Removed.